### PR TITLE
wolfssl: refer to it as wolfSSL only

### PR DESCRIPTION
**Pull request by bagder**
Remove support for, references to and use of "cyaSSL" from the source
and docs. wolfSSL is the current name and there's no point in keeping
references to ancient history.

### CODE REVIEW

File: docs/libcurl/opts/CURLINFO_TLS_SSL_PTR.3 [hunk 1]
```diff
@@ -83,9 +83,9 @@ mbedtls_ssl_context *
 ssl_context *
 .IP "Secure Channel"
 CtxtHandle *
-.IP "Secure Transport (DarwinSSL)"
+.IP "Secure Transport"
 SSLContext *
-.IP "WolfSSL (formerly CyaSSL)"
+.IP "WolfSSL"
```
Comment (danielgustafsson) [inline docs/libcurl/opts/CURLINFO_TLS_SSL_PTR.3]: wolfSSL? (ie capitalization)

File: docs/libcurl/opts/CURLOPT_SSL_CTX_DATA.3 [hunk 2]
```diff
@@ -112,8 +112,8 @@ int main(void)
 }
 .fi
 .SH AVAILABILITY
-Added in 7.11.0 for OpenSSL. Added in 7.42.0 for wolfSSL/CyaSSL. Other SSL
-backends not supported.
+Added in 7.11.0 for OpenSSL. Added in 7.42.0 for wolfSSL. Other SSL backends
+not supported.
```
Comment (danielgustafsson) [inline docs/libcurl/opts/CURLOPT_SSL_CTX_DATA.3]: Should we reword this while at it?
> Other SSL backends do not support this option.

File: docs/libcurl/opts/CURLOPT_SSL_CTX_FUNCTION.3 [hunk 3]
```diff
@@ -72,8 +71,8 @@ See cacertinmem.c in docs/examples directory for usage example.
 
 https://curl.haxx.se/libcurl/c/cacertinmem.html
 .SH AVAILABILITY
-Added in 7.11.0 for OpenSSL. Added in 7.42.0 for wolfSSL/CyaSSL. Added in
-7.54.0 for mbedTLS. Other SSL backends not supported.
+Added in 7.11.0 for OpenSSL. Added in 7.42.0 for wolfSSL. Added in 7.54.0 for
+mbedTLS. Other SSL backends not supported.
```
Comment (danielgustafsson) [inline docs/libcurl/opts/CURLOPT_SSL_CTX_FUNCTION.3]: Same as above comment.

File: docs/libcurl/opts/CURLOPT_SSL_CTX_DATA.3 (hunk 2 above)
Comment (bagder) [reply docs/libcurl/opts/CURLOPT_SSL_CTX_DATA.3]: any particular suggestion?

Comment (captain-caveman2k) [reply docs/libcurl/opts/CURLOPT_SSL_CTX_DATA.3]: How about:

Added in 7.11.0 for OpenSSL and 7.42.0 for wolfSSL. Other SSL backends are
not supported.

File: docs/libcurl/opts/CURLOPT_SSL_CTX_FUNCTION.3 (hunk 3 above)
Comment (captain-caveman2k) [reply docs/libcurl/opts/CURLOPT_SSL_CTX_FUNCTION.3]: And similarly:

Added in 7.11.0 for OpenSSL, 7.42.0 for wolfSSL and 7.54.0 for mbedTLS. Other SSL backends are not supported.

**Pull request closed without merging**